R/abm.R

```diff
@@ -143,7 +150,7 @@ abm_table2 <- function(con = con_bib(), unit_code, analysis_start = abm_config()
 #' @param last the largest integer in the range to use
 #' @param width the desired width of intervals
 #' @return data frame with label for each interval and one row for each year
-
+#' @export
 sliding_intervals <- function(first, last, width){
```

Comment:
Not important, I guess, but this fcn could return a tibble since we're doing that elsewhere?

Reply:
Seems reasonable. :) Do we have a general "to do when we have time to refactor" issue somewhere?